### PR TITLE
Add Kotlin variant for LayoutSpec

### PR DESCRIPTION
**Pull request by pavlospt**
Add Kotlin variant for `LayoutSpec`. More to follow 🎉

### CODE REVIEW

File: docs/_docs/layout-specs.md [hunk 1]
```diff
@@ -5,59 +5,8 @@ layout: docs
...
-The second child is a [Text](/javadoc/com/facebook/litho/widget/Text) component that takes a prop named `text` and fills the remaining horizontal space available in `MyComponent` by using `grow(1f)` (equivalent to Android's `layoutWeight` from `LinearLayout`). The text size is defined in `my_text_size` dimension resource.
-
-```java
-Text.create(c)
-    .text(title)
-    .textSizeRes(R.dimen.my_text_size)
-    .grow(1f)
-```
-You can check the full [Yoga](https://facebook.github.io/yoga/docs/learn-more/) documentation to see all the layout features that the framework exposes.
+{{layout-spec-java | markdownify }}
```
Comment (passy) [inline docs/_docs/layout-specs.md]: nit: Could you add spaces before and after the element? I.e. `{{ layout-spec-java ... }`

**Comment by passy** [comment]
Okay, it seems like the switcher is implemented via a custom layout in `getting-started`. It sits in `layouts/

**Comment by pavlospt** [comment]
Let me fix those issues and get back. I wanted to check if everything is okay with that one and follow with the rest. Only issue is i am not sure how to test it before committing. Jekyll local has issues with SCSS.

**Comment by pavlospt** [comment]
@passy Ok now I have broken the changes properly! Added `LayoutSpec` variations. Let me know if you want me to add more variations on that same PR or open a separate one for each of them, when this is merged, in ordered to keep changes contained!

**Comment by pavlospt** [comment]
@passy Ok so when this is merged It is fine to proceed with a PR per section with Java/Kotlin variants, right?

**Pull request closed without merging**